sky/execution.py

```diff
@@ -120,8 +120,9 @@ def _execute(
       dag: sky.Dag.
       dryrun: bool; if True, only print the provision info (e.g., cluster
         yaml).
-      teardown: bool; whether to teardown the launched resources after
-        execution.
+      terminate: bool; whether to terminate the launched resources after
+        execution. If idle_minutes_to_autostop is set, the cluster will be
```

Comment:
nit: after the execution of the task exits (successfully or abnormally)

I believe it's cleaned up after abnormal job exits, just double-checking.

Reply:
Yes, it will clean up after the job fails, but if the cluster fails at the setup/filemount. 
During adding this I found that it previously will not terminate the cluster when the user ctrl-c'ed during the logging for the execution. I just moved the teardown into the `finally: ...` block to make sure the clean up will happen after the ctrl-c. Wdyt?